tests/tensor/test_basic.py

```diff
@@ -828,11 +828,8 @@ def check(dtype, N, M_=None, k=0):
         # allowed.
         if M is None and config.mode in ["DebugMode", "DEBUG_MODE"]:
             M = N
-        N_symb = iscalar()
-        M_symb = iscalar()
-        k_symb = iscalar()
-        f = function([N_symb, M_symb, k_symb], eye(N_symb, M_symb, k_symb, dtype=dtype))
-        result = f(N, M, k)
+
+        result = eye(N, M, k, dtype).eval()
```

Comment:
Any reason to change the previous test?

Reply:
The structure of the function is different. The new function takes python integers, whereas the previous function took symbolic tensors. It's not necessary, then, to compile eye -- it can be directly evaluated.

This change might be undesirable?

Reply:
The old test still runs, I guess I got too excited. I will revert this change.

Reply:
Yeah, I don't see anything that doesn't work with symbolic inputs.

Reply:
This has now been reverted.